plugins/woocommerce/tests/e2e/README.md

```diff
@@ -186,6 +186,15 @@ For example:
 - `PUPPETEER_SLOWMO=10` - will run tests faster
 - `PUPPETEER_SLOWMO=70` - will run tests slower
 
+### How to retry failed tests
```

Comment:
I'm posting some follow up comments because I think everyone who regularly works in this repo would be happy with retries being enabled by default. We can update this to reflect that.